### PR TITLE
ffsend: add page

### PR DESCRIPTION
**Pull request by NargiT**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/ffsend.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# ffsend
+
+> Easy and securely share files from command line.
```
Comment (agnivade) [inline pages/linux/ffsend.md]: s/Easy/Easily. Although there may be a better grammatical construction for this. @sbrl ?

File: pages/linux/ffsend.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+`ffsend upload {{file}} -p {{password}}`
+
+- Download a file protected by password:
+
+`ffsend download {{file}} -p {{password}}`
+
+- Upload a file and allow 4 downloads:
+
+`ffsend upload {{file}} -d 4`
```
Comment (agnivade) [inline pages/linux/ffsend.md]: 4 in tokens (`{{}}`)

File: pages/linux/ffsend.md (hunk 1 above)
Comment (NargiT) [reply pages/linux/ffsend.md]: fixed

File: pages/linux/ffsend.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# ffsend
+
+> Easily and securely share files from command line.
+
```
Comment (sbrl) [inline pages/linux/ffsend.md]: Perhaps we could add a line here like this:

```markdown
> Homepage: <https://gitlab.com/timvisee/ffsend>.
```

Comment (NargiT) [reply pages/linux/ffsend.md]: added

**Pull request merged**